cime_config/testdefs/testlist_clm.xml

```diff
@@ -1989,6 +1989,7 @@
     <machines>
       <machine name="izumi" compiler="intel" category="aux_clm"/>
       <machine name="derecho" compiler="nvhpc" category="aux_clm"/>
+      <machine name="derecho" compiler="nvhpc" category="ssp"/>
```

Comment:
Ahhh, adding a new testlist for ssp. That sounds interesting.

I do suggest that you use intel or gnu here though, as we know there are problems with nvhpc.